src/report_workflow/README.md

```diff
@@ -0,0 +1,24 @@
+#### Generate test-report manifest for each test.
+The input requires a mandatory `test-manifest-path`, `--artifact-paths`, `--base-path`, `--test-type` and `--test-run-id`, optional `--output-path` and `--component` to automatically generate the test-report manifest after testing.
```

Comment:
nit: Add `<required>` and `<optional>` in the below table so that all information related to params is in one place

Reply:
Added.

Reply:
You can remove this from here and instead add more description what this workflow actually does. 
```suggestion
As the name specifies, the test report workflow helps to automatically generate a consolidated report of the tests run at distribution level along with commands to reproduce the error and associated failures.
These tests include integration tests. BWC tests.
```